config/default.toml

```diff
@@ -44,7 +44,7 @@ DefaultEmailHost = "mozilla.com"
     # Triggers are one of ["first", "second", "third", "final", "ignore"]
 
     [Events.DataDog]
-        Enabled = false
+        Enabled = true
```

Comment:
Oops. This shouldn't be the case (but is fine). It just means that statistics reporting is on.